resources/carrier/en/234.txt

```diff
@@ -176,6 +186,7 @@
 2347782|Starcomms
 2347783|Starcomms
 2347784|Starcomms
+234800|Alpha Technologies
```

Comment:
Please take a loot at first comment. 700 is validated as UAN and 800 is validated as toll-free. If you think that these are mobile ranges ? Please raise an issue and share any official evidence you may have. See our [guidelines](https://github.com/googlei18n/libphonenumber/blob/master/CONTRIBUTING.md#provide-supporting-evidence).